pkg/collector/corechecks/ebpf/ebpf.go

```diff
@@ -5,6 +5,7 @@
 
 //go:build cgo && linux
 
+// Package ebpf implements the ebpf check.
```

Comment:
This package actually contains all the ebpf-based checks

Reply:
Fixed. Thanks :+1: